Gemfile

```diff
@@ -87,7 +87,7 @@ eval_gemfile("./Gemfile.local") if File.exist?("./Gemfile.local")
 if RUBY_PLATFORM.match?(/mswin|mingw|windows/)
   instance_eval do
     ruby_exe_dir = RbConfig::CONFIG["bindir"]
-    assemblies = Dir.glob(File.expand_path("distro/ruby_bin_folder", Dir.pwd) + "/*.dll")
+    assemblies = Dir.glob(File.expand_path("distro/ruby_bin_folder", File.dirname(__FILE__)) + "/*.dll")
```

Comment:
```suggestion
    assemblies = Dir.glob(File.expand_path("distro/ruby_bin_folder", __dir__) + "/*.dll")
```

Reply:
I was gonna start using this a lot but then ryan davis suggested that __dir__ was a terrible thing, now i'm like ???

Reply:
Hmm, I'm curious what his reasoning is. It's no biggie to stick with what's there, just less parents and things.

Reply:
rubocop has a rule to enforce it (although it doesn't seem to do anything, i'm very confused, but i can't actually find anything against it now).  think i'm leaning towards using `__dir__` now and we can always undo that later if its some kind of giant mistake....